docs/source/release-notes.rst

```diff
@@ -42,7 +42,7 @@ were an attempt to make a forward-looking guarantee that was entirely unneeded.
 
 Altogether, this has been an extremely productive period for Hyper-h2, and a
 lot of great work has been done by the community. To that end, we'd also like
-to extend a great thankyou to those contributers who made their first contribution
+to extend a great thankyou to those contributors who made their first contribution
```

Comment:
I thought about that. At least for non-US English the noun 'thankyou' is one word. [\[ref\]](http://www.future-perfect.co.uk/grammar-tip/is-it-thank-you-thankyou-or-thank-you/).